docs/01-Introduction.md

```diff
@@ -1,23 +1,34 @@
 # Introduction
 
-The idea behind testing is not new. You can't sleep well if you are not confident that your last commit didn't take down the whole application.
+The idea behind testing is not new. You can't sleep well if you are not confident
+that your last commit didn't take down the whole application.
 Having your application covered with tests gives you more trust in the stability of your application. That's all.
 
-In most cases tests don't guarantee that the application works 100% as it is supposed to. You can't predict all possible scenarios and exceptional situations for complex apps, but with tests you can cover the most important parts of your app and at least be sure they work as predicted.
+In most cases tests don't guarantee that the application works 100% as it is supposed to.
+You can't predict all possible scenarios and exceptional situations for complex apps,
+but with tests you can cover the most important parts of your app and at least be sure they work as predicted.
 
-There are plenty of ways to test your application. The most popular paradigm is [Unit Testing](http://en.wikipedia.org/wiki/Unit_testing). As for web applications, testing the controller, or model in isolation doesn't prove your application is working. To test the behavior of your application as a whole, you should write functional or acceptance tests.
+There are plenty of ways to test your application.
+The most popular paradigm is [Unit Testing](http://en.wikipedia.org/wiki/Unit_testing).
+For web applications, testing the controller, or model in isolation doesn't prove your application is working.
```

Comment:
For web applications, testing just the controller and / or the model doesn't prove that your application is working.